shared/actions/json/teams.json

```diff
@@ -37,7 +37,7 @@
     "getTeamPublicity": {
       "teamname": "string"
     },
-    "haveChosenChannelsForTeam": {
+    "addTeamWithChosenChannels": {
```

Comment:
i found these names confusing, since `have` isn't a verb, and it's a set of teamnames...hopefully these are better names?